integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_automatically_saved_pages_row.html

```diff
@@ -0,0 +1,36 @@
+{% extends "../_todo_dashboard_row.html" %}
+{% load i18n %}
+{% block todo_dashboard_icon %}
+    save
+{% endblock todo_dashboard_icon %}
+{% block todo_dashboard_title_link %}
+    {% url 'pages' region_slug=request.region.slug language_slug=default_language_slug %}
```

Comment:
> Another (probably not very important) problem is that there seems to be no way to get rid of an autosave that should not get published.

@david-venhoff what do you mean by this? You can reject unwanted autosaves in the version view, right? Then it will revert to the latest public version which should result in the autosave no longer being present on the dashboard.

Reply:
Ah, I was not aware of that. Then you can ignore this point ^^